### PR TITLE
Add licensing.

### LINKED ISSUE #43: Add license and third-party-copyright info following move to separate repo
The Zulip server repo's `docs/THIRDPARTY` file has the following information about the files that were moved from there into this repo:
```
Files: api/*
Copyright: 2012-2017 Dropbox, Inc., Kandra Labs, Inc., and contributors
License: Expat

Files: api/integrations/perforce/git_p4.py
Copyright: 2007 Simon Hausmann <simon@lst.de>,
 2007 Trolltech ASA
License: Expat
Comment: https://raw.github.com/git/git/34022ba/git-p4.py

Files: bots/jabber_mirror_backend.py
Copyright: 2013 Permabit, Inc., 2013-2014 Dropbox, Inc.
License: Expat
```

Now that this code is in its own repo, we should move that information here, and also add a LICENSE file with the Expat license to help make that part clear.

### PR DESCRIPTION
**Pull request by roberthoenig**
Fixes #43.

I'm not experienced with licenses, so this should probably be reviewed with extra care.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Copyright 2017 Zulip
```
Comment (derAnfaenger) [inline LICENSE]: Is 2017 right here?

Comment (rht) [reply LICENSE]: I think it should have been 2012-2017.

Moreover, is Zulip right here? Should it have been "Zulip developers and contributors" (as an alternative to putting the name of the orgs/co's)

Comment (derAnfaenger) [reply LICENSE]: I agree - the main repo reads "Copyright 2011-2017 Dropbox, Inc., Kandra Labs, Inc., and contributors", so maybe we should use this?

**Comment by rht** [comment]
Hmmm, @zulipbot, as a contributor, I am clearly not in the set of {Zulip maintainer}. Maybe it should be "someone reviewed ..." when the reviewer is not a maintainer?

**Comment by timabbott** [comment]
@lonerz @synicalsyntax that's a fair point on the zulipbot "reviewed" notifications

**Comment by lonerz** [comment]
@timabbott I guess, we can use the same idea as labelling. There's a specific group of people where a review will switch "needs review" to "reviewed".

**Comment by zulipbot-test** [comment]
Hello @roberthoenig, a Zulip maintainer reviewed this pull request over 10 days ago, but you haven't updated your pull request since. Please take a look at the requested changes and update your pull request accordingly.

Thank you for your valuable contributions to Zulip!

**Comment by zulipbot-aayush** [comment]
Hello @roberthoenig, a Zulip maintainer reviewed this pull request over 10 days ago, but you haven't updated your pull request since. Please take a look at the requested changes and update your pull request accordingly.

Thank you for your valuable contributions to Zulip!

**Comment by timabbott** [comment]
I merged this as 1dc2bf8ba6c6511d445a69e2ee9b6d2069b4fd29 after some cleanup.

**Pull request closed without merging**